kernel/values.ml

```diff
@@ -0,0 +1,21 @@
+(************************************************************************)
```

Comment:
Please use modules_without_implementation and a .mli instead of a .ml without .mli

Reply:
Do we do this for all such modules? I had the impression we had some ml files without mli files in Coq to emulate this (because at some point it was bad or something).

Reply:
At some point = before dune maybe
We don't do it for some modules but that's no reason to add more

Reply:
Done anyways.